requirements.txt

```diff
@@ -1,12 +1,194 @@
-backoff
-joblib<=1.3.2
-openai>=0.28.1,<2.0.0
-pandas
-regex
-ujson
-tqdm
-datasets
-requests
-optuna
-pydantic==2.5.0
-structlog
+## This file will be deprecated and won't be maintain after 2024-06-01 ##
+## We are not using this file for building our package and it is just for reference and development of people who are using this file ##
+
+## note: When you are adding new dependencies, please add them to the requirements.txt file too and update that till ##
```

Comment:
Add the command that was used to generate the req.txt file

Reply:
done.